packages/flutter/test/services/text_input_test.dart

```diff
@@ -90,3 +143,79 @@ void main() {
     });
   });
 }
+
+class FakeTextInputClient implements TextInputClient {
+  const FakeTextInputClient();
+
+  TextInputConfiguration get configuration => const TextInputConfiguration();
+
+  @override
+  void performAction(TextInputAction action) => throw UnimplementedError();
+
+  @override
+  void updateEditingValue(TextEditingValue value) => throw UnimplementedError();
+  @override
```

Comment:
uber-nit: add an empty line above this one for constancy?

Reply:
Done.